src/index.ts

```diff
@@ -328,9 +343,11 @@ ACTIONS (can be repeated, in any order)
     -b, --filterBands  {0|1|2|3}            Strip spherical-harmonic bands > N
 
 GLOBAL OPTIONS
-    -w, --overwrite                         Overwrite output file if it already exists
+    -w, --overwrite                         Overwrite output file if it already exists. Default is false.
     -h, --help                              Show this help and exit
     -v, --version                           Show version and exit
+    -g, --gpu                               Enable gpu for k-means clustering. Specify --no-gpu to disable. Default is true.
```

Comment:
Why do we need this option if it default to true? Don't we just need no-gpu?

Reply:
I didn't go with no-gpu because it makes the code a bit messy, but I've changed it.